pkg/internalapis/cnsvolumeoperationrequest/config/cns.vmware.com_cnsvolumeoperationrequests.yaml

```diff
@@ -0,0 +1,152 @@
+                      and may not correspond to the task invocation timestamp on CNS.
+                    format: date-time
+                    type: string
+                  taskStatus:
+                    description: TaskStatus describes the current status of the task
+                      invoked on CNS. Valid strings are "In Progress", "Successful"
+                      and "Failed".
+                    type: string
+                required:
+                - taskId
+                - taskInvocationTimestamp
+                type: object
+              latestOperationDetails:
+                description: LatestOperationDetails stores the details of the latest
+                  operations performed on the volume. Should have a maximum of 10
+                  entries.
+                items:
+                  description: OperationDetails stores the details of the operation
+                    performed on a volume.
+                  properties:
+                    error:
+                      description: Error represents the error returned if the task
+                        fails on CNS. Defaults to empty string.
+                      type: string
+                    opId:
+                      description: OpID stores the OpID for a task that was invoked
+                        on CNS for a volume.
+                      type: string
+                    taskId:
+                      description: TaskID stores the task for an operation that was
+                        invoked on CNS for a volume.
+                      type: string
+                    taskInvocationTimestamp:
+                      description: TaskInvocationTimestamp represents the time at
+                        which the task was invoked. This timestamp is derived from
+                        the cluster and may not correspond to the task invocation
+                        timestamp on CNS.
+                      format: date-time
+                      type: string
+                    taskStatus:
+                      description: TaskStatus describes the current status of the
+                        task invoked on CNS. Valid strings are "In Progress", "Successful"
+                        and "Failed".
+                      type: string
+                  required:
+                  - taskId
+                  - taskInvocationTimestamp
+                  type: object
+                type: array
+              snapshotID:
+                description: SnapshotID is the unique ID of the backend snapshot.
+                  Populated during successful CreateSnapshot calls.
+                type: string
+              volumeID:
+                description: VolumeID is the unique ID of the backend volume. Populated
+                  during successful CreateVolume calls.
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field. Please remove the subresources section